lib/display/display.ex

```diff
@@ -19,13 +19,18 @@ defmodule Inky.Display do
 
   @spec spec_for(:impression) :: Inky.Display.t()
   def spec_for(type = :impression) do
+    width = 600
+    height = 448
+
     %__MODULE__{
       type: type,
-      width: 600,
-      height: 448,
-      packed_resolution: <<2, 88, 1, 192>>, # I used the struct.pack in Python to generate this
+      width: width,
+      height: height,
+      packed_resolution:
+        <<width::unsigned-big-integer-size(16)>> <> <<height::unsigned-big-integer-size(16)>>,
```

Comment:
This produces the same result as before (<<2, 88, 1, 192>>), but makes it a little more clear what's happening. The Pimoroni Python library uses `struct.pack(">HH", self.width, self.height))` to accomplish this and the `">HH"` arg passed literally translates into two 16 bit unsigned big endian integers.